### ISSUE TITLE
[Blog] How the REST and GraphQL API docs are published

### ISSUE THREAD

**Issue by Aritra8438** [layer5io/layer5#4775]
Contributors and community members are encouraged to post on https://layer5.io/blog. A new blog post fitting the following description is requested to be written by a community member:

**Desired Title:**  How the Meshery REST and GraphQL API docs are published?

**Topic:** Will give a clear understanding of the prerequisites, swagger, openAPI and how github workflow helps publish the API docs automatically using Swagger.

**Core points to convey in the post:** 
1. OpenAPI and Swagger
2. Github workflows
3. How they help to publish the REST and GraphQL API docs
---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="16px" align="left" /><h3>Contributor Resources</h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

**Comment by Ghat0tkach** [comment]
hey @Aritra8438 
let's discuss about your progress on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)
https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Comment by Ghat0tkach** [comment]
@Aritra8438 any progress?

**Comment by Aritra8438** [comment]
No, it will take a little bit of time to get back to this! Sorry for that!

**Comment by abhijeetgauravm** [comment]
@Aritra8438 updates here?

**Comment by sandramsc** [comment]
This issue has been open for some time with no recent activity, unassigning to open it up for new contributors to give it a go.

**Comment by narasimha-1511** [comment]
i want to work on this
